auth/simple.go

```diff
@@ -70,3 +117,40 @@ func (a simpleAuth) GetUser(ctx context.Context) (string, error) {
 	}
 	return id.(string), nil
 }
+
+func (a simpleAuthN) authenticate(userID string) error {
+	if userID == "" {
+		return fmt.Errorf("Must provide header: [%s]", a.userIDHeader)
+	}
+	return nil
+}
+
+func (a simpleAuthZ) authorize(httpMethod, userID string) error {
+	switch httpMethod {
+	case "GET", "":
```

Comment:
Use `http.MethodGet` consts. Also when will the method be empty?

Reply:
Yeah makes sense to block empty request methods